huaweicloud/services/dli/resource_huaweicloud_dli_queue.go

```diff
@@ -180,7 +180,14 @@ func resourceDliQueueCreate(d *schema.ResourceData, meta interface{}) error {
 	d.SetId(queueName)
 
 	// This is a workaround to avoid issue: the queue is assigning, which is not available
-	time.Sleep(120 * time.Second) //lintignore:R018
+	time.Sleep(4 * time.Minute) //lintignore:R018
```

Comment:
Why sleep? 
Can we use waitfor according to the state?

Reply:
No api returns this information